os/binfmt/libelf/libelf.h

```diff
@@ -437,6 +437,18 @@ void elf_delete_bin_section_addr(uint8_t bin_idx);
 #ifdef CONFIG_BINFMT_SECTION_UNIFIED_MEMORY
 void *elf_find_start_section_addr(struct binary_s *binp);
 #endif
+
+/****************************************************************************
+ * Name: get_bin_addr_list
+ *
+ * Description:
+ *   Return the pointer to the bin info address list
+ *
+ * Returned Value:
+ *   Return the pointer to the bin info address list
```

Comment:
Could you differentiate this from the description please?
You could phrase it just as "Pointer to the bin info address list"